configs/foreman_118.yaml

```diff
@@ -0,0 +1,140 @@
+:project: foreman
+:prior_releases:
+  :1.17.0:
+    :redmine_release_id: 296
+  :1.17.1:
+    :redmine_release_id: 297
+:releases:
+  :1.18.0:
+    :redmine_release_id: 330
+:repos:
+  :foreman:
+    :main: true
+    :branch: 1.18-stable
```

Comment:
This is likely a bit confusing, when this is an actual branch the following has to be included:

```
version_branch: true
```

Otherwise it treats it as a tag.